src/symbols/Node/Node.module.css

```diff
@@ -1,7 +1,8 @@
 .rect {
   fill: #2b2c3e;
-  stroke: transparent;
   transition: stroke 100ms ease-in-out;
+  /* TODO Why duplicated stroke? */
```

Comment:
@amcdnl This seems to be a mistake, should I remove `stroke: transparent;`? Or is it a CSS trick that needs to be documented?

Reply:
Good catch - its a mistake - can you remove the first one?

Reply:
I'll do.